docs/guide/writing-tasks.md

```diff
@@ -392,6 +392,21 @@ a relation between an encrypted variable and a repository for which the encrypte
       CREDENTIALS: SECURED[!qwerty] # won't be decrypted in any case
     ```
 
+!!! tip "Skipping Task in Forked Repository"
```

Comment:
Let's switch this and two blocks above from `!!!` to `???` so they all are collapsable.

Reply:
You mean like this? (I don't know how/where to check results in the way how they display on your site, so I'm just blindly changing the files)

I know I'm kinda tunnel-visioning my own use case and experience, so I have not enough "distance" to decide what should be the final look of your docs, I just kick-started this with the PR to see if it leads somewhere. I.e. I fully expect you to keep tossing this around until it fully fits your liking of end result. :)